### DISCUSSION TITLE
alternative copy/paste keybindings

### DISCUSSION THREAD

**Discussion by alikve** [Jelmerro/Vieb#545] [Ideas]
I'd like to ask for alternative copy/paste keybindings for Linux could be added:
`["<C-v>", "<S-Insert>"]`
`["<C-c>", "<C-Insert>"]`
`["<C-x>", "<S-Del>"]`

**Comment by Jelmerro** [comment]
Do you mean that these keybindings are missing in the input bar? Because if so, I agree that they should be added indeed.

**Reply by alikve** [reply]
Yes, they are missing. I added them to my local build (to `app/renderer/input.js`) and encountered no problems.

**Reply by Jelmerro** [reply]
Added in https://github.com/Jelmerro/Vieb/commit/8f77b251a7c7ae3210c87f8359aae1a332ac8adf, will be included in the next release.

**Reply by Jelmerro** [reply]
Implemented in 12.0.0 :tada:
